docs/fix/empty-wheelbarrows.rst

```diff
@@ -0,0 +1,33 @@
+fix/empty-wheelbarrows
+======================
+
+.. dfhack-tool::
+    :summary: Empties stuck items from wheelbarrows
```

Comment:
needs to end in a period to be parsed by the in-game help engine.

Reply:
Added period.